### PR TITLE
merging in upstream 0.8

### PR DESCRIPTION
**Pull request by mjburghoffer**

### CODE REVIEW

File: gwizard-swagger/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# GWizard Swagger
+
+
+GWizard Swagger servess a swagger configuration file generated from your JAX-RS
```
Comment (christydennison) [inline gwizard-swagger/README.md]: serves*

File: gwizard-swagger/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# GWizard Swagger
+
+
+GWizard Swagger servess a swagger configuration file generated from your JAX-RS 
+resources and @Api annoted methods. At the moment, only the swagger json or yaml
```
Comment (christydennison) [inline gwizard-swagger/README.md]: annotated*

File: gwizard-swagger/pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		<version>0.8.riq.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>gwizard-swagger</artifactId>
+	<name>GWizard Swagger</name>
+	<packaging>jar</packaging>
+	
+	<dependencies>
+
+		<dependency>
```
Comment (christydennison) [inline gwizard-swagger/pom.xml]: indent?

File: gwizard-swagger/src/main/java/org/gwizard/swagger/SwaggerModule.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+@Slf4j
+@EqualsAndHashCode(callSuper=false, of={})	// makes installation of this module idempotent
+public class SwaggerModule extends ServletModule {
+	
+	@Override
+	protected void configureServlets() {
+		log.debug("Configuring swagger guice module...");
+		Multibinder.newSetBinder(binder(), ServletContextListener.class).addBinding().to(SwaggerServletContextListener.class);
+		bind(ApiListingResource.class);
+    bind(SwaggerSerializers.class);
```
Comment (christydennison) [inline gwizard-swagger/src/main/java/org/gwizard/swagger/SwaggerModule.java]: indent?

File: gwizard-swagger/README.md (hunk 1 above)
Comment (mjburghoffer) [inline gwizard-swagger/README.md]: lol this is me merging in upstream changes.

File: gwizard-swagger/src/main/java/org/gwizard/swagger/SwaggerModule.java (hunk 4 above)
Comment (mjburghoffer) [inline gwizard-swagger/src/main/java/org/gwizard/swagger/SwaggerModule.java]: again, i am merely merging in upstream changes.

**Pull request merged**